base/bitarray.jl

```diff
@@ -600,7 +600,6 @@ function insert!(B::BitVector, i::Integer, item)
 end
 
 function _deleteat!(B::BitVector, i::Integer)
-
     k, j = get_chunks_id(i)
```

Comment:
This makes the code less readable, as it detaches the contents of the function from the surrounding blocks that denote where the function starts and ends. a consistent policy is better than an inconsistent one, even if aesthetic judgements about it differ between contributors.